packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -96,18 +88,14 @@ export function GetDetail({
                     </Text>
                   </Box>
                 </Box>
-                <Box
-                  display="flex"
-                  flexDirection="column"
-                  gap="4"
-                  onClick={() => hasMobileCompanionApp && getMobileWallet(id)}
-                  {...(!hasMobileCompanionApp && downloadUrls?.browserExtension
-                    ? linkProps(downloadUrls.browserExtension)
```

Comment:
I noticed that in some scenarios we were rendering an `a` element around a `button`, so this PR also cleans this up, passing the link props to `ActionButton` instead.